### DISCUSSION TITLE
AAAA / IPv6 record support

### DISCUSSION THREAD

**Discussion by ghost** [is-a-dev/register#4854] [Q&A] [answered]
Hi!

Just got my .is-a.dev subdomain, and I wanted to thank you! I also had a thought - is AAAA record support going to be added soon? I know that the web is moving closer to IPv6, so it might be cool to see if it gets adopted by is-a.dev!

**Comment by wdhdev** [comment, marked as answer]
I'm not sure whether we are going to support IPv6 yet, but we may add support for ot soon.

**Reply by ghost** [reply]
Okay, that sounds great! I have another question - if is-a.dev ever shuts down, will we get a little bit of notice so we can move our sites & other content over to different platforms? I really appreciate the free service, but I'm a bit afraid of having the proverbial rug pulled from under me.

**Reply by wdhdev** [reply]
@bsnts We have no intentions of shutting the service down as it is pretty cheap to maintain (~$50/year), but if there was some bizarre reason and we shut down the service we would most likely give notice.

**Reply by ghost** [reply]
Ah, that sounds pretty good.

**Reply by wdhdev** [reply]
If this answered your question, please mark it as the answer.

**Reply by ghost** [reply]
> If this answered your question, please mark it as the answer.

I don't know if this discussion category - "Ideas" supports answers. Maybe you need to enable that? I have no idea.

**Reply by wdhdev** [reply]
> > If this answered your question, please mark it as the answer.
> 
> I don't know if this discussion category - "Ideas" supports answers. Maybe you need to enable that? I have no idea.

I fixed the category
